### DISCUSSION TITLE
입체주의

### DISCUSSION THREAD

**Discussion by giscus** [padosum/blog#33] [General]
# 입체주의

입체주의 (Cubism)   아비뇽의 처녀들 -파블로 피카소   파블로 피카소가 회화는 자연을 재현하는 것이란 생각을 입체주의로 깨부쉈다.  분석적 입체주의 종합적 입체주의  파피에 콜레 

http://localhost:8000/wiki/Cubism/

**Comment by padosum** [comment]
🤖 giscus test
